src/test/java/uk/gov/justice/hmpps/prison/executablespecification/BulkAppointmentsStepDefinitions.java

```diff
@@ -120,8 +120,8 @@ private static LocalDateTime asLocalDateTime(String string) {
 
     private static Map<String, String> replaceRelativeTimes(Map<String, String> map) {
         final var result = new HashMap<>(map);
-        result.computeIfPresent("startTime", BulkAppointmentsStepDefinitions::asDateTimeString);
-        result.computeIfPresent("endTime", BulkAppointmentsStepDefinitions::asDateTimeString);
+        result.compute("startTime", BulkAppointmentsStepDefinitions::asDateTimeString);
```

Comment:
were getting `null` coming through from jackson instead of empty string so need to replace with `""` for the test